Makefile

```diff
@@ -1,10 +1,30 @@
 PORT ?= 8080
+IMAGE_TAG ?= latest
```

Comment:
```suggestion
IMAGE_TAG ?= main
```

Reply:
The pipeline will set tag using main and commit sha. Using non-official "latest" is a way to avoid polluting the official images when running locally